pr_agent/tools/ticket_pr_compliance_check.py

```
@@ -101,36 +105,129 @@ async def extract_tickets(git_provider):
+                                                "ticket_url": f"{jira_url}/browse/{subtask.key}",
+                                                "title": getattr(subtask_issue.fields, "summary", ""),
+                                                "body": subtask_body,
+                                            }
+                                        )
+                                    except JIRAError as e_sub:
+                                        get_logger().warning(
+                                            f"Failed to fetch Jira subtask {subtask.key} for {key}: {e_sub.text}"
+                                        )
+                                    except Exception as e_sub_other:
+                                        get_logger().warning(
+                                            f"An unexpected error occurred fetching Jira subtask {subtask.key}: {e_sub_other}"
```

Ruff (0.11.9)

Line too long (126 > 120)

(E501)